qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -249,4 +249,10 @@
             variants:
                 - from_old_installer:
                     update_from_previous_installer = yes
+                    static_ip = 1.2.3.4
```

Comment:
How about making those private ip addreses more reasonable, such as 192.168.x.xx or 10.10.xx.xx?

Reply:
It's all ok for me, because the main checkpoint is if the static ip can be saved after upgrade the driver.
I just follow current polarion case's cofiguration, if @heywji  also agree to use 192..., I will modify the code.Thanks!

Reply:
I have no problem with this making. Both are fine.